### ISSUE TITLE
Publish the helm chart to GHCR

### ISSUE THREAD

**Issue by stefanprodan** [weaveworks/weave-gitops#2300]
As a Flux user I want to install the Weave GitOps UI from GitHub Container Registry instead of using Weaveworks Helm Registry. As a cluster admin, I don't want to depend on the weave.works uptime as there is no SLA for that, instead I want to pull the charts from ghcr.io.

PS. Pushing helm charts to GHCR is an easy task, here is how podinfo does it with GH Actions:

```yaml
      - name: Publish Helm chart to GHCR
        run: |
          helm package charts/podinfo
          helm push podinfo-${{ steps.prep.outputs.VERSION }}.tgz oci://ghcr.io/stefanprodan/charts
```

I suggest that Weaveworks created a `ghcr.io/weaveworks/charts` repo to host the `weave-gitops` chart.

**Comment by ozamosi** [comment]
Start with making sure that publishing the chart works - this is easiest done in `ghcr.io/weaveworks/weave-gitops`.

Before publishing instructions to end users, decide if we want to move it to `weaveworks/charts`.

**Comment by chanwit** [comment]
After some trials and errors, I found that `ghcr.io/weaveworks/charts` was a pseudo name, not a real repo name.
The steps I did locally were:
  * `cd charts/gitops-server`
  * `helm package .`
  * `helm push weave-gitops-2.0.10.tgz oci://ghcr.io/weaveworks/charts`
  * Then I jumped to GitHub, link ghcr.io/weaveworks/charts/weave-gitops:2.0.10 to the `weaveworks/weave-gitops` repo.
  * Set the package to be public.
